src/palace/manager/api/axis.py

```diff
@@ -549,21 +564,32 @@ def patron_activity(
         pin: str | None,
         identifier: Identifier | None = None,
         internal_format: str | None = None,
+        debug_info: list[str] | None = None,
     ) -> list[AxisLoanInfo | HoldInfo]:
         if identifier:
             assert identifier.identifier is not None
             title_ids = [identifier.identifier]
         else:
             title_ids = None
+
         availability = self.availability(
             patron_id=patron.authorization_identifier, title_ids=title_ids
         )
-        return list(
+
+        availability_content_str = availability.content.decode("utf-8")
+        if debug_info:
+            debug_info.append(f"arguments to availability call: title_ids={title_ids}")
+            debug_info.append(
+                f"response to availability call: status={availability.status_code}, content={availability_content_str}"
+            )
+        loan_info_list = list(
             AvailabilityResponseParser(self, internal_format).process_all(
-                availability.content
+                availability_content_str
```

Comment:
Minor: The generic `process_all` method does take `bytes` or `str` so this change should be fine. All the tests for `AvailabilityResponseParser.process_all` pass in `bytes` via the `sample_data` method. Since you are changing what we pass into the function it might be worth updating the tests to match, so we are testing the same input type as the method gets in real usage.

https://github.com/ThePalaceProject/circulation/blob/3ea99eed70bf6a48bcd7cfd9121dfa4e997f65dd/tests/manager/api/test_axis.py#L1504-L1614